source/material_model/interface.cc

```diff
@@ -288,7 +290,6 @@ namespace aspect
         }
     }
 
-
     template <int dim>
     MaterialModelInputs<dim>::MaterialModelInputs(const FEValuesBase<dim,dim> &fe_values,
                                                   const typename DoFHandler<dim>::active_cell_iterator *cell_x,
```

Comment:
It would of course be nice to also make this into a reference rather than a pointer. how many places would this affect?

In the interest of keeping review simple, maybe keep that for a follow-up patch if you want to do it. Or open an issue about it.